src/tokens/border-tokens/border-tokens.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import tokens from '../transformed.tokens.json';
+import BorderTokens from "./border-tokens";
+
+const meta = {
+  title: 'Tokens/Borders',
+  component: BorderTokens,
+  parameters: {
+    layout: 'fullscreen',
+  },
```

Comment:
Is there a particular reason to add this? It removes the padding from the border to the content, making it (in my opinion) a bit harder to read.